src/client/dfs/dfs.c

```diff
@@ -846,16 +850,44 @@ open_symlink(dfs_t *dfs, daos_handle_t th, dfs_obj_t *parent, int flags,
 	return ENOTSUP;
 }
 
+static void
+set_daos_iod(bool create, daos_iod_t *iod, char *buf, size_t size)
+{
+	d_iov_set(&iod->iod_name, buf, strlen(buf));
+	iod->iod_nr	= 1;
+	iod->iod_size	= DAOS_REC_ANY;
+	iod->iod_recxs	= NULL;
+	iod->iod_type	= DAOS_IOD_SINGLE;
+
+	if (create) {
+		iod->iod_size = size;
+	}
+}
+
+static void
+create_sb(bool create, daos_iod_t *iods, daos_key_t *dkey)
```

Comment:
the name of this function is misleading.
you are not creating the sb, but just setting the dkey, akey, & IOD array.
there is already a function called open_sb with a create flag, so you can see how this can get confusing.
can be renamed to something like: set_inode_params() ?
and also maybe rename the bool variable to: for_update

Reply:
Done, function and parameter were renamed